### DISCUSSION TITLE
QEMU/KVM User session

### DISCUSSION THREAD

**Discussion by linebullet** [virt-manager/virt-manager#795] [General]
Hi, how i can start virt-manager with user privelegy? Have Permission denied.
![macos](https://github.com/user-attachments/assets/36429efb-4b3f-414f-87d5-7591d77e631a)

**Comment by phrdina** [comment]
Hi, the issue here is that your home directory `/home/kali` allows only the user `kali` to access that directory. I see that you've changed the `user` and `group` config in `/etc/libvirt/qemu.conf`, in order for that option to take effect you need to restart `libvirtd` or `virtqemud` process.

To answer your question, you need to add a new connection into virt-manager and select `QEMU/KVM User Session` as the hypervisor.
